CONTRIBUTING.md

```diff
@@ -65,6 +65,21 @@ The messages that appear in our changelogs are defined using changesets.
 
 [Click here to learn what changesets are, and how to add one.](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md)
 
+### Adding a changeset
+Don't forget to also include a changeset, by running this command at the root of the project:
+
+```sh
+yarn changeset
```

Comment:
@AladinoBorges 

This repo has a [package-lock.json](https://github.com/devopness/devopness/blob/main/package-lock.json#L814) in its root, and not a ~yarn.lock~. 
For this reason I believe our command examples should use `npm` instead of `yarn`. Or maybe even simpler, `npx` to execute directly from the command line without needing to install first?

```suggestion
npx changeset
```